src/cmdlib.sh

```diff
@@ -842,6 +845,9 @@ EOF
     fi
     rc="$(cat "${rc_file}")"
 
+    # cleanup tail before nuking dir containing file it's following
+    kill "$tail_pid"
```

Comment:
I think there's a potential race here where `tail` could be killed before it finished actually printing all the output, even though qemu already exited. A simple fix is to just e.g. `sleep 1` or whatever but ughhh. Really wish we could go back to the virtio-serial approach.

Reply:
And indeed: https://github.com/openshift/os/pull/1498#issuecomment-2097383950